### PR TITLE
fix(app): Add the support to scan the qr code without pincode (otp)

### LINKED ISSUE #209: Add the support to scan the qr code without pincode (otp)

### PR DESCRIPTION
**Pull request by talwinder50**
closes #209

Signed-off-by: Talwinder kaur <talwinder.kaur@avast.com>

https://user-images.githubusercontent.com/7862595/214351970-65450d05-7a24-4750-bb50-d35c74bd7341.mov

### CODE REVIEW

**Review by rolsonquadras — APPROVED** [review]

**Review by vkubiv — APPROVED** [review]
please wait until my changes is merged

**Comment by rolsonquadras** [comment]
> please wait until my changes is merged

@talwinder50 - Can you please rebase as we have merge conflict now?

**Pull request merged**
